src/executables/csql.c

```diff
@@ -1455,6 +1459,195 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
 	}
 
       break;
+
+    case S_CMD_CONNECT:
+      /*dbname_cpy can be stored DB name+ @ + remote hostname + \0 */
+      char dbname_cpy[MAXLOGNAME + CUB_MAXHOSTNAMELEN + 2];
```

Comment:
MAXLOGNAME define is not appropriate.

Reply:
@airnet73 
I used MAXLOGNAME because when we create db, the name length is limited by MAXLOGNAME.
So, do we need to make other macro like below?
`#define DB_NAME_LEN MAXLOGNAME`

Reply:
Yes, it would be nice to add a macro.
I'm not sure if 64 is the right value for DB_NAME.
#define DB_NAME_LEN (64)

Reply:
As we knew that MAXLOGNAME is 16. So, we set the DB_NAME_LEN is 16.

Thank you. I will apply it.

Reply:
@airnet73 
I decide this macro because digits (16) is not easy to figure out the reason.
But below macro is easier than 16.
#define DB_NAME_LEN MAXLOGNAME